src/components/include/utils/logger.h

```diff
@@ -33,151 +33,110 @@
 #ifndef SRC_COMPONENTS_INCLUDE_UTILS_LOGGER_H_
 #define SRC_COMPONENTS_INCLUDE_UTILS_LOGGER_H_
 
+#include "utils/ilogger.h"
+
 #ifdef ENABLE_LOG
-#include <errno.h>
-#include <log4cxx/propertyconfigurator.h>
-#include <log4cxx/spi/loggingevent.h>
-#include <string.h>
 #include <sstream>
+#include <string>
+
 #include "utils/auto_trace.h"
 #include "utils/logger_status.h"
-#include "utils/push_log.h"
-#endif  // ENABLE_LOG
 
-#ifdef ENABLE_LOG
-
-#define CREATE_LOGGERPTR_GLOBAL(logger_var, logger_name) \
-  namespace {                                            \
-  CREATE_LOGGERPTR_LOCAL(logger_var, logger_name);       \
+#define SDL_CREATE_LOGGERPTR(component_name)  \
+  namespace {                              \
+  static std::string logger_(component_name); \
   }
 
-#define CREATE_LOGGERPTR_LOCAL(logger_var, logger_name) \
-  log4cxx::LoggerPtr logger_var =                       \
-      log4cxx::LoggerPtr(log4cxx::Logger::getLogger(logger_name));
+#define SDL_CREATE_LOCAL_LOGGERPTR(component_name) \
```

Comment:
@VladSemenyuk not obvious name. It just defines string, not any kind of pointer. Suggest to rename